tools/autograd/derivatives.yaml

```diff
@@ -1461,11 +1461,8 @@
 - name: _svd_helper(Tensor self, bool some, bool compute_uv) -> (Tensor U, Tensor S, Tensor V)
   self: svd_backward(grads, self, some, compute_uv, U, S, V)
 
-- name: symeig(Tensor self, bool eigenvectors=False, bool upper=True) -> (Tensor eigenvalues, Tensor eigenvectors)
-  self: eigh_backward(grads, self, eigenvectors, eigenvalues, eigenvectors_return)
-
 - name: linalg_eigh(Tensor self, str UPLO="L") -> (Tensor eigenvalues, Tensor eigenvectors)
-  self: eigh_backward(grads, self, /*eigenvectors=*/true, eigenvalues, eigenvectors)
```

Comment:
This is going to conflict when you rebase as https://github.com/pytorch/pytorch/pull/70528 was merged.
You'll need to remove the `/*symeig_eigenvector=*/eigenvectors)` argument from the function signature and the logic it encapsulates in the `eig_backward` function itself.